media-ranker/app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,50 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.order(rank: :desc)
+  end
+
+  def create
+    album = Album.create(album_params)
+    album.update_attribute(:rank, 1)
+    redirect_to albums_path
```

Comment:
:information_source: Be sure to check out polymorphic paths and see if you know how you could implement them. They're perfectly suited for this project.